.github/workflows/ci.yml

```diff
@@ -45,6 +45,12 @@ jobs:
       - name: Lint with flake8
         run: |
           flake8  --max-line-length=127
+      - name: Formatting check with black
+        run: |
+          black . --check
+      - name: Type check with mypy
+        run: |
+          mypy .
```

Comment:
`black` is for prettier formatting, I've been using it for a while.
`mypy` is for type checking